frontend/src/scenes/session-recordings/components/AiRegexHelper/aiRegexHelperLogic.ts

```diff
@@ -0,0 +1,87 @@
+import { actions, kea, listeners, path, reducers } from 'kea'
+import api from 'lib/api'
+import { lemonToast } from 'lib/lemon-ui/LemonToast/LemonToast'
+import posthog from 'posthog-js'
+
+import type { aiRegexHelperLogicType } from './aiRegexHelperLogicType'
+
+export const aiRegexHelperLogic = kea<aiRegexHelperLogicType>([
+    path(['lib', 'components', 'AiRegexHelper', 'aiRegexHelperLogic']),
+    actions({
+        setIsOpen: (isOpen: boolean) => ({ isOpen }),
+        setInput: (input: string) => ({ input }),
+        handleGenerateRegex: () => ({}),
```

Comment:
you can do 
```suggestion
        handleGenerateRegex: true,
```
as shorthand for an action that takes no params